### ISSUE TITLE
After autoupdate and then auto open DS4Windows (fist time), Chinese language (Traditional, Taiwan) will not work

### ISSUE THREAD

**Issue by SyaoranChang** [Ryochan7/DS4Windows#2129]
Hi,

Since after update to DS4Windows 3.0 or before.
After autoupdate and then auto open DS4Windows (fist time), Chinese language (Traditional, Taiwan) will not work.
Close and reopen again, then Chinese language will work fine.

![DS4Windows BUGSnipaste_2021-05-07_03-06-08](https://user-images.githubusercontent.com/18550228/117692091-3f2ac600-b1ef-11eb-988e-49cb3b86f108.png)

**Comment by Ryochan7** [comment]
Maybe the DS4Updater routine needs to be changed to explicitly set the current working directory for the process before it launches DS4Windows. Assembly loading changed with .NET 5 and that has affected the loading of the translation assembly files. DS4Windows has to be launched with the install folder acting as the current working directory. Explorer and Desktop Shortcuts do that.

The original .NET 5 port used a package called NetCoreBeauty (IIRC) to work around that requirement. I decided against using that package though as I did not like the output folder structure and it added more DLL dependencies.
